src/client/src/orangehrmPerformancePlugin/pages/ReviewSearch.vue

```diff
@@ -0,0 +1,399 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+<template>
+  <div class="orangehrm-background-container">
+    <oxd-table-filter :filter-title="$t('performance.review_list')">
+      <oxd-form @submitValid="filterItems" @reset="filterItems">
+        <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+          <oxd-grid-item>
+            <employee-autocomplete
+              v-model="filters.employee"
+              :params="{
+                includeEmployees: 'currentAndPast',
+              }"
+            />
+          </oxd-grid-item>
+          <oxd-grid-item>
+            <jobtitle-dropdown v-model="filters.jobTitle" />
+          </oxd-grid-item>
+          <oxd-grid-item>
+            <oxd-input-field
+              v-model="filters.status"
+              type="select"
+              :label="$t('performance.review_status')"
+              :clear="false"
+              :options="statusOpts"
```

Comment:
@Super-Chama there is an extra option here for the status. Should I add a prop to status dropdown component made previously or remove it?

Reply:
Whats the extra option? :thinking:

Reply:
inactive status. it's only shown in manage review screen and not review list

Reply:
by "remove it" I meant the component not the option

Reply:
I think you should keep the component since its been used elsewhere. you can even override just the option prop
`<review-status-dropdown v-model="filters.status" :options="statusOpts" />`